manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.7.0
+PackageLocale: ru-RU
+Publisher: OneScript
```

Comment:
```suggestion
Publisher: 1Script Team (Open Source)
```

Reply:
should I also move manifest to `1/1Script Team (Open Source)/OneScript/` path ?

Reply:
Na there is no need for that :)